R/rank_regression.R

```diff
@@ -1,55 +1,53 @@
 #'
 #' For the Weibull, estimates are additionally transformed such that they are in
-#' line with the parameterization provided by the \emph{stats} package
-#' (see \link[stats]{Weibull}).
+#' line with the parameterization provided by the *stats* package
+#' (see [Weibull][stats::Weibull]).
 #'
 #' @details
-#' If \code{distribution} is \code{"weibull"} or \code{"weibull3"}, the approximated
+#' If `distribution` is `"weibull"` or `"weibull3"`, the approximated
 #' confidence intervals for the parameters can only be estimated on the following
-#' confidence levels (see 'References' \emph{(Mock, 1995)}):
-#' \itemize{
-#'   \item \code{conf_level} = 0.90,
-#'   \item \code{conf_level} = 0.95,
-#'   \item \code{conf_level} = 0.99.
-#' }
+#' confidence levels (see 'References' *(Mock, 1995)*):
+#'
+#' * `conf_level = 0.90`
+#' * `conf_level = 0.95`
+#' * `conf_level = 0.99`
 #'
 #' If the distribution is not the Weibull, the confidence intervals of the
 #' parameters are computed on the basis of a heteroscedasticity-consistent
 #' covariance matrix. Here it should be said that there is no statistical foundation
-#' to determine the standard errors of the parameters using \emph{Least Squares}
-#' in context of \emph{Rank Regression}. For an accepted statistical method use
-#' \link[=ml_estimation]{maximum likelihood}.
+#' to determine the standard errors of the parameters using *Least Squares*
+#' in context of *Rank Regression*. For an accepted statistical method use
+#' [maximum likelihood][ml_estimation].
 #'
-#' @param x Object of class \code{wt_cdf_estimation} returned from
-#'  \code{\link{estimate_cdf}}.
+#' @param x A `tibble` of class `wt_cdf_estimation` returned by [estimate_cdf].
 #' @param distribution Supposed distribution of the random variable.
-#' @param conf_level Confidence level of the interval. If \code{distribution} is
-#'   \code{"weibull"} this must be one of \code{0.9}, \code{0.95} or \code{0.99}.
+#' @param conf_level Confidence level of the interval. If `distribution` is
+#'   `"weibull"` this must be one of `0.9`, `0.95` or `0.99`.
 #' @template dots
 #'
 #' @template return-rank-regression
-#' @templateVar data A tibble with class \code{wt_cdf_estimation} returned from \code{\link{estimate_cdf}}.
+#' @templateVar data A `tibble` with class `wt_cdf_estimation` returned by [estimate_cdf].
 #' @return
-#'   If more than one method was specified in \code{\link{estimate_cdf}},
-#'   the resulting output is a list with class \code{wt_model_estimation_list}.
-#'   In this case each list element has classes \code{wt_rank_regression} and
-#'   \code{wt_model_estimation} and the items listed above, are included.
+#'   If more than one method was specified in [estimate_cdf], the resulting output
+#'   is a list with class `wt_model_estimation_list`. In this case each list element
+#'   has classes `wt_rank_regression` and `wt_model_estimation` and the items listed
+#'   above, are included.
```

Comment:
Punctuation:
In this case, each list element has classes `wt_rank_regression` and `wt_model_estimation`, and the items listed above are included.